internals/templates/store.test.js

```diff
@@ -3,7 +3,7 @@
  */
 
 import expect from 'expect';
-import configureStore from './store';
+import configureStore from '../store'; // eslint-disable-line
```

Comment:
eslint is disabled for this line as ../store is invalid when store.test.js is located in templates as a sibling of store.js. Hence ./store which satisfied eslint.

When copied into app/tests it is a level below and ./store fails to resolve.

Disabling eslint for the line seemed the most expedient solution. Really, IMO, the templates folder should mirror the app folder structure more completely.